packages/blade/src/components/Input/BaseInput/BaseInputTagSlot.native.tsx

```diff
@@ -52,6 +53,26 @@ const ScrollableTagSlotContainer = ({
   );
 };
 
+const ClickableText = ({
+  children,
+  handleOnClick,
+}: {
+  children: StringChildrenType;
+  handleOnClick: BaseInputTagSlotProps['handleOnClick'];
+}): React.ReactElement => {
+  return (
+    <TouchableWithoutFeedback
+      onPress={() => {
+        handleOnClick?.({ name: '', value: '' });
```

Comment:
Why is this going as blank? Shouldn't we send nothing instead?

Reply:
let me see if we can actually send the correct values here. Reliably sending value might be tricky since in AutoComplete and Select, inputs in themselves don't know what value is. We should be able to send the correct name though. 

I am maintaining a todo list of these task. Will create a followup PR with fix for all this